docs/backend/server_arguments.md

```diff
@@ -21,6 +21,7 @@ python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3-8B-Instruct
 ```
 
 - See [hyperparameter tuning](hyperparameter_tuning.md) on tuning hyperparameters for better performance.
+- For docker and Kubernetes runs, you need to set up shared memory which is used for communication between processes. See `--shm-size` for docker and `/dev/shm` size update for Kubernetes manifests.
```

Comment:
good summary. but it seem this doc `server_arguments.md` is majorly for CLI (python run) use case.
if we aim for "docker"/"k8s", maybe this file  is not suitable?
https://github.com/sgl-project/sglang/blob/main/docs/start/install.md

Reply:
also, as you wrote, in Docker part of https://github.com/sgl-project/sglang/blob/main/docs/start/install.md shared memory is already mentioned. And in Kubernetes section there's just a link to manifest, so I'm suggesting: 
- to modify single node manifest
- to update the docs in the place where `--tp` is actually mentioned as server argument